client/asset/eth/eth.go

```diff
@@ -4683,7 +4683,7 @@ func (w *baseWallet) checkPendingTxs() {
 			// on it, cancel that request.
 			if pendingTx.actionRequested {
 				pendingTx.actionRequested = false
-				w.requestAction(asset.ActionResolved, pendingTx.ID, nil, pendingTx.TokenID)
+				w.resolveAction(pendingTx.ID, pendingTx.TokenID)
```

Comment:
So this is the root cause of this issue, my understanding is that's just a typo of sorts